pom.xml

```
@@ -121,7 +121,7 @@
     <mockito.version>3.6.28</mockito.version>
     <jersey.version>2.33</jersey.version>
     <!-- must align with kafka version https://github.com/charithe/kafka-junit -->
-    <kafka-junit.version>4.2.1</kafka-junit.version>
+    <kafka-junit.version>4.2.2</kafka-junit.version>
```

Check with this latest version. If it works, let's update kafka to 3.2.1
```suggestion
    <kafka-junit.version>4.2.3</kafka-junit.version>
```

---

Kafka 3.2.1 can be used with Kafka Junit 4.2.3. Thanks.